spec/v2/index.bs

```diff
@@ -443,7 +442,7 @@ Advisement:
         <td><dfn>fossilCarbonContent</dfn> : [=Decimal=]
         <td>String
         <td>M
-        <td>The fossil carbon amount embodied in the product. The value MUST be calculated per <{CarbonFootprint/declaredUnit|declared unit}> with unit `kg of CO2 equivalent per declared unit` (`kgCO2e / declaredUnit`), expressed as a [=decimal=] equal to or greater than zero.
+        <td>The fossil carbon amount embodied in the product. The value MUST be calculated per <{CarbonFootprint/declaredUnit|declared unit}> with unit `kg per declared unit` (`kg / declaredUnit`), expressed as a [=decimal=] equal to or greater than zero.
```

Comment:
I could be entirely wrong, but I am not sure this change from kg CO2 equivalent to kg CO2 was intended methodologically, I can't find the email trail or note about this. I know we had to change this for biogenicCarbonContent, but this was already done. I did find however that we had an open ask that Cecilia updates the description of the field, so I sent her an email asking for this (we have had a lot of questions about fossil emissions, hence the benefit to update this).

Reply:
That said, either way we should keep in the reference to kg of CO2, just remove "equivalent" in case she confirms this should be done. I fear that if we remove the full string "of CO2" we are removing specificity that is actually useful, that this is indeed intended as kg of CO2 (or kg of emissions), depending on the answer from Cecilia.

Reply:
Ignore me! I just found the email trail on this and realized I was confusing this field with fossilGhgEmissions! My apologies. Indeed the change is correct as you have done it Martin, only recommendation is just to keep "kg of CO2" rather than remove CO2 entirely, for clarity sake.